criu/files.c

```diff
@@ -1335,6 +1335,8 @@ int prepare_fds(struct pstree_item *me)
 
 static int fchroot(int fd)
 {
+	if (!has_cap_sys_chroot(opts.cap_eff))
+		return 0;
```

Comment:
Hmmm. I don't think that it is a good idea to return 0 if we can't do chroot. I think we need to inspect each place where we call fchroot and decide what we need to do if we don't have caps.

Reply:
Can you be more specific what changes you want. Not sure what to do here?

Reply:
fchroot can't return 0 if it has done nothing. I think we need to find all places where it is called and see when we can skip it in the unprivileged mode...